tests/api/transactions/GET.test.js

```diff
@@ -77,4 +80,18 @@ describe("GET /api/transactions", () =>{
       expect(res.status).toEqual(200);
     });
   });
+  describe("user id does not have a transaction that pertain to it", ()=>{
+    it("should return error object", async() =>{
```

Comment:
If the user id doesn't have any transactions, shouldn't we return an empty list instead?

Reply:
Change to `it("should return an empty list", ...`